pkg/operator/controller/ingress/status.go

```diff
@@ -725,6 +728,10 @@ func checkDefaultCertificate(secret *corev1.Secret, domain string) error {
 		if cert.Subject.CommonName == domain && !foundSAN {
 			return fmt.Errorf("certificate in secret %s/%s has legacy Common Name (CN) but has no Subject Alternative Name (SAN) for domain: %s", secret.Namespace, secret.Name, domain)
 		}
+		switch cert.SignatureAlgorithm {
+		case x509.SHA1WithRSA, x509.ECDSAWithSHA1:
+			return fmt.Errorf("certificate in secret %s/%s has weak SHA1 signature algorithm: %s\nsee https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-networking for more details", secret.Namespace, secret.Name, cert.SignatureAlgorithm)
```

Comment:
Did you mean to link https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-sha-haproxy-support-removed_release-notes ?  I don't see any `ocp-4-16-networking` anchors in the release notes at the moment, and while I do see things like `ocp-4-16-networking_release-notes`, I don't see a benefit to linking there vs. linking the specific HAProxy vs. SHA-1 section.

Reply:
Ah good catch. I could have sworn I tested this, I wonder if it changed recently. I'll use the specific anchor, and I guess we'll hope that doesn't change again. Done.

Reply:
yeah, looks like openshift/openshift-docs@bdd9453e84a789d6ef2 brought in the more-specific anchor in April, and openshift/openshift-docs@0d46a75d67313d9fd5f01a80ff60d38487fec1ec broke your old anchor a few days ago 😅